ShareGISData/GPTools/arcpy/dlaFieldCalculator.py

```diff
@@ -19,6 +19,84 @@
 # ---------------------------------------------------------------------------
 
 import os, sys, traceback, time, arcpy,  dla
+import collections
```

Comment:
Did not test this, probably want to move to a new class.  Just put it here for now @SteveGrise